# What: Remove the deprecated XC_GGA_X_HERMAN functional for which
#       support was dropped in libxc 6.1.0
#
# Author: Sven Hansen

diff -ruN gamess.orig/source/libxc.src gamess/source/libxc.src
--- gamess.orig/source/libxc.src	2024-07-23 08:20:28.000000000 +0200
+++ gamess/source/libxc.src	2025-11-07 11:35:45.841431921 +0100
@@ -2161,7 +2161,7 @@
     call add_free_functional(GGA_X_group,GGA_X_GAM_name,XC_GGA_X_GAM)
     call add_free_functional(GGA_X_group,GGA_X_GG99_name,XC_GGA_X_GG99)
     call add_free_functional(GGA_X_group,GGA_X_HCTH_A_name,XC_GGA_X_HCTH_A)
-    call add_free_functional(GGA_X_group,GGA_X_HERMAN_name,XC_GGA_X_HERMAN)
+    !call add_free_functional(GGA_X_group,GGA_X_HERMAN_name,XC_GGA_X_HERMAN)
     call add_free_functional(GGA_X_group,GGA_X_HJS_B88_name,XC_GGA_X_HJS_B88)
     call add_free_functional(GGA_X_group,GGA_X_HJS_B88_V2_name,XC_GGA_X_HJS_B88_V2)
     call add_free_functional(GGA_X_group,GGA_X_HJS_B97X_name,XC_GGA_X_HJS_B97X)
